lib/facade_converter.ts

```diff
@@ -176,6 +173,7 @@ export class FacadeConverter extends base.TranspilerBase {
     var fileName = decl.getSourceFile().fileName;
     fileName = this.getRelativeFileName(fileName);
     fileName = fileName.replace(/(\.d)?\.ts$/, '');
+    fileName = fileName.replace(FACADE_NODE_MODULES_PREFIX, '');
```

Comment:
makes more sense in this location where we were already patching up the file path